data/definitions.cpp

```diff
@@ -59,6 +59,5 @@ void Definitions::addImportedType(const QString &typeName)
 const TypeAssignment *Definitions::type(const QString &name) const
 {
     const auto it = m_typeByNameMap.find(name);
-    QTC_ASSERT(it != m_typeByNameMap.end(), return nullptr);
```

Comment:
that was probably only `QTC_ASSERT` usage in whole file - maybe include could be dropped as well